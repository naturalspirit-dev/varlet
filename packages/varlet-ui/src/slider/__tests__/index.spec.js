const Slider = require('../../../cjs/slider').default
const { render } = require('@testing-library/vue')

test('test slider', async () => {
	const wrapper = render(Slider)
	console.log(wrapper)
})
