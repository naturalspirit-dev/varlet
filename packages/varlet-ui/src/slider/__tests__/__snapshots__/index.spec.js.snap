// Vitest Snapshot v1

exports[`test slider rules prop 1`] = `
"<div class=\\"var-slider\\">
  <div class=\\"var-slider__block\\">
    <div class=\\"var-slider__track\\">
      <div class=\\"var-slider__track-background\\"></div>
      <div class=\\"var-slider__track-fill\\" style=\\"width: 0%; left: 0%;\\"></div>
    </div>
    <div class=\\"var-slider__thumb\\" style=\\"left: 0%;\\">
      <div class=\\"var-slider__thumb-block\\"></div>
      <div class=\\"var-slider__thumb-ripple\\"></div>
      <div class=\\"var-slider__thumb-label\\"><span>0</span></div>
    </div>
  </div>
  <transition-stub class=\\"var-slider__form\\" var-slider-cover=\\"\\">
    <!--v-if-->
  </transition-stub>
</div>"
`;

exports[`test slider rules prop 2`] = `
"<div class=\\"var-slider\\">
  <div class=\\"var-slider__block\\">
    <div class=\\"var-slider__track\\">
      <div class=\\"var-slider__track-background\\"></div>
      <div class=\\"var-slider__track-fill\\" style=\\"width: 0%; left: 0%;\\"></div>
    </div>
    <div class=\\"var-slider__thumb\\" style=\\"left: 0%;\\">
      <div class=\\"var-slider__thumb-block\\"></div>
      <div class=\\"var-slider__thumb-ripple\\"></div>
      <div class=\\"var-slider__thumb-label\\"><span>NaN</span></div>
    </div>
  </div>
  <transition-stub class=\\"var-slider__form\\" var-slider-cover=\\"\\">
    <!--v-if-->
  </transition-stub>
</div>"
`;

exports[`test slider style 1`] = `
"<div class=\\"var-slider\\">
  <div class=\\"var-slider__block\\" style=\\"height: 24px; margin: 0px 4px;\\">
    <div class=\\"var-slider__track\\">
      <div class=\\"var-slider__track-background\\" style=\\"background: rgb(58, 104, 180); height: 4px;\\"></div>
      <div class=\\"var-slider__track-fill\\" style=\\"width: 30%; left: 0%; background: rgb(224, 115, 44);\\"></div>
    </div>
    <div class=\\"var-slider__thumb\\" style=\\"left: 30%;\\">
      <div class=\\"var-slider__thumb-block\\" style=\\"background: rgb(226, 82, 65); height: 8px; width: 8px;\\"></div>
      <div class=\\"var-slider__thumb-ripple\\" style=\\"background: rgb(226, 82, 65); height: 0px; width: 0px;\\"></div>
      <div class=\\"var-slider__thumb-label\\" style=\\"background: purple; color: rgb(237, 237, 237); height: 16px; width: 16px;\\"><span>30</span></div>
    </div>
  </div>
  <transition-stub class=\\"var-slider__form\\" var-slider-cover=\\"\\">
    <!--v-if-->
  </transition-stub>
</div>"
`;
